pkg/panels/blackbox/blackbox.go

```diff
@@ -0,0 +1,777 @@
+package blackbox
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	"github.com/perses/community-dashboards/pkg/promql"
+	commonSdk "github.com/perses/perses/go-sdk/common"
+	"github.com/perses/perses/go-sdk/panel"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	stat "github.com/perses/perses/go-sdk/panel/stat"
+	timeSeriesPanel "github.com/perses/perses/go-sdk/panel/time-series"
+	"github.com/perses/perses/go-sdk/prometheus/query"
+)
+
+// ProbeStatusMapfunc creates a panel option for displaying Blackbox Probe Success for all instances
+//
+// The panel uses the following Prometheus metrics:
+// - probe_success: indicates if the probe succeeded
+//
+// The panel shows:
+// - Probe success, either 1 if up, or 0 if down
+//
+// Parameters:
+//   - datasourceName: The name of the Prometheus data source.
+//   - labelMatchers: A variadic parameter for Prometheus label matchers to filter the query.
+//
+// Returns:
+//   - panelgroup.Option: A panel option that can be added to a panel group.
+
+func ProbeStatusMapfunc(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
```

Comment:
I think we can remove that func suffix, wdyt?

Reply:
thanks, removed, resolving